### PR TITLE
HashiCorp Vault - add the folder arg to the configure-engine cmd

### PR DESCRIPTION
**Pull request by Itay4**
## Status
- [x] Ready

## Does it break backward compatibility?
   - [x] No

## Must have
- [x] Documentation

### CODE REVIEW

File: Packs/HashiCorp-Vault/Integrations/HashiCorpVault/HashiCorpVault.py [hunk 1]
```diff
@@ -172,11 +172,13 @@ def list_secrets_command():
...
 
 
-def list_secrets(engine_path, version):
+def list_secrets(engine_path, version, folder=None):
     path = engine_path
 
     if version == '2':
         path += '/metadata'
+        if folder:
+            path += '/' + folder
```
Comment (liorblob) [inline Packs/HashiCorp-Vault/Integrations/HashiCorpVault/HashiCorpVault.py:181]: Make sure `folder` does not already contain a `/` or use `path.join`

**Review by liorblob — APPROVED** [review]
Nice 1 comment

File: Packs/HashiCorp-Vault/Integrations/HashiCorpVault/HashiCorpVault.py (hunk 1 above)
Comment (Itay4) [reply Packs/HashiCorp-Vault/Integrations/HashiCorpVault/HashiCorpVault.py]: good catch - added

**Comment by Itay4** [comment]
build fails on server issue non related to the changes in this pull request
merging

**Pull request merged**